easyconfigs/r/RELION/RELION-5.0-beta-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '5.0-beta'
+local_commit = '90d239eda870d64834e642aaa19ce3e6d0ab5ab0' # There is no release yet, so we use the latest commit (08/01/2024 15:09)
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/3dem/%(namelower)s/archive/']
+sources = [local_commit + '.tar.gz']
+patches = ['RELION-5.0-beta_cuda_capabilities.patch']
+#checksums = [
+#    'aedc0c7e1806094274d8442680aade810943648b08875148f5da226121753a0e',  # RELION-5.0_beta_cuda_capabilities.patch
+#]
+
+
+builddependencies = [('CMake', '3.24.3')]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),    
+    ('X11', '20220504'),
+    ('FLTK', '1.3.8'),
+    ('LibTIFF', '4.3.0'),
+]
+
+# lowest supported capability is 3.5
+cuda_compute_capabilities = ['3.5', '3.7', '5.0', '5.2', '5.3', '6.0', '6.1', '6.2', '7.0']
```

Comment:
The A100 has compute capability 8.0, so may be worth adding that one as well (if supported by Relion).

Reply:
Well spotted :) The manual simply says 3.5+, so I'll add 8.0 and rebuild